lib/screens/app/ecosystem/overview.dart

```diff
@@ -37,6 +39,21 @@ class _OverviewState extends State<Overview> {
     ]);
   }
 
+  void onBuy() {
+    UrlLauncher.launch("https://www.joinseeds.com/buy-seeds.php");
+  }
+
+  void onDonate() {
+    NavigationService.of(context).navigateTo(
+      Routes.transferForm,
+      TransferFormArguments(
+        "balisupport.org",
+        "kaelaatleewo",
+        "https://balisupport.org/wp-content/uploads/2020/04/Asset-12@3x-1.png",
+      ),
+    );
+  }
+
```

Comment:
This should be a remote config in Firebase so we can add different causes for donations @7flash 

Should not be hard coded in the code base